.github/workflows/build.yaml

```diff
@@ -0,0 +1,102 @@
+---
+name: Project Air UI
```

Comment:
Maybe we can discuss this on Monday, but since we will have effectively 2 different angular apps in the same repository (1 for on-prem, 1 for cloud). Do we need 2 different workflows? Or multiple jobs in the same workflow? Something to think about as we might want to decouple the testing and building.